ADO.NET/OAuthSampleApp/README.md

```diff
@@ -0,0 +1,29 @@
+## Overview
+
+The Vertica ADO.NET driver now allows connecting through OAuth access tokens. 
+
+This sample application was built in .NET 6.0 and can be run in .NET Framework 4.6+ or .NET Core 3.1+.
+
+# Prerequisites
+
+For information on installing .NET, starting a Vertica docker container, and how to reference a NuGet package, see the README.md in the ADO.NET base directory.
```

Comment:
I have the same comments in this line as in the duplicate from the other readme.

Reply:
Fixed.